src/style.css

```diff
@@ -0,0 +1,108 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir.ttc);
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 600;
```

Comment:
Not critical, but why 600? `Heavy` font weight is usually 900 (can be 800)
600 - is something like `bold`

https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#common_weight_name_mapping